patches/minecraft/net/minecraft/data/advancements/AdvancementProvider.java.patch

```diff
@@ -1,53 +1,10 @@
 --- a/net/minecraft/data/advancements/AdvancementProvider.java
 +++ b/net/minecraft/data/advancements/AdvancementProvider.java
-@@ -18,8 +_,19 @@
+@@ -14,6 +_,7 @@
+ import net.minecraft.data.PackOutput;
+ import net.minecraft.resources.ResourceLocation;
+ 
++// Forge: Use ForgeAdvancementProvider instead, provides ease of access for the ExistingFileHelper in the generator
+ public class AdvancementProvider implements DataProvider {
```

Comment:
Shouldnt this have  proper javadoc comment with deprecation instead of just a single line comment?

Reply:
If we push people twards a new version, we tend to deprecate.

Reply:
Ok, changing it now then.

Reply:
Done